src/main/java/controllers/ServletController.java

```diff
@@ -0,0 +1,97 @@
+package controllers;
+
+import calculator.Calculator;
+import calculator.WebCalculatorFactory;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import static controllers.ControllerConstants.ABS_RANGE;
+import static controllers.ControllerConstants.DIVIDE;
+import static controllers.ControllerConstants.EXPRESSION;
+import static controllers.ControllerConstants.LOCATION;
+import static controllers.ControllerConstants.MINUS;
+import static controllers.ControllerConstants.MISSING_EXPRESSION;
+import static controllers.ControllerConstants.MULTIPLY;
+import static controllers.ControllerConstants.OVER_RANGE;
+import static controllers.ControllerConstants.PLUS;
+import static controllers.ControllerConstants.WRONG_EXPRESSION;
+
+public class ServletController implements Controller {
+
+    private static final WebCalculatorFactory FACTORY = new WebCalculatorFactory();
+    private static final Calculator CALCULATOR_SERVICE = FACTORY.createCalculator();
+
+    public void getResult(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        HttpSession session = request.getSession();
+        PrintWriter writer = response.getWriter();
+
+        try {
+            int result = CALCULATOR_SERVICE.calculate(session);
+            writer.print(result);
+            response.setStatus(HttpServletResponse.SC_OK);
+            writer.close();
+        } catch (Exception e) {
+            response.sendError(HttpServletResponse.SC_CONFLICT, MISSING_EXPRESSION);
+        }
+    }
+
+
+    public void deleteData(HttpServletRequest request, HttpServletResponse response) {
+        HttpSession session = request.getSession();
+        String attributeName = request.getRequestURI().substring(6);
```

Comment:
Please move it to separate method. Also it is contains anti-pattern magic numbers. move digits to constant